scripts/test_imm.sh

```diff
@@ -17,9 +17,9 @@ TESTS_NUMBER=0
 PASSED_TESTS=0
 JPEGS_INFERENCE_ACCURACY=0
 
-SERVER_CERT="../helm-deployment/management-api-subchart/certs/server-tf.crt"
-CLIENT_CERT="../helm-deployment/management-api-subchart/certs/client-tf.crt"
-CLIENT_KEY="../helm-deployment/management-api-subchart/certs/client-tf.key"
+SERVER_CERT="$HELM_INSTALL_DIR/management-api-subchart/certs/server-tf.crt"
```

Comment:
i guess now we need HELM_INSTALL_DIR env in circleci cfg

Reply:
this variable is already set up by installer script, but maybe we should default it to ../helm-deployment if empty. Another, better option is to generate client certs during tenant creation, and fetch server certificate using ./get_ca_cert.sh  script. This change would separate the ./test_imm.sh from helm deployment location

Reply:
second option, to generate client certs and use get_ca_cert is really good, i think we should do that

Reply:
@beataszelag Would you like to do it ?